src/components/SearchForm.tsx

```diff
@@ -18,30 +18,37 @@ export default class SearchForm extends Component<
 		value: this.props.initSearchQuery,
 	};
 
-	handleOnChange = (value: string): void => {
-		this.setState(() => ({
-			value,
-		}));
-	};
+	form: HTMLFormElement | null = null;
 
 	handleCallback = (e: React.FormEvent): void => {
 		e.preventDefault();
 		this.props.onSearch(this.state.value);
 	};
 
-	render() {
-		const placeholder = "What do you want to watch?";
+	handleKeyDown = (e: React.KeyboardEvent) => {
```

Comment:
Not an error. But. If I recall it right "enter" will call "onSubmit" function from form. Why do we need this here?

Reply:
You're right and that was my assumption during the initial implementation at the first task. But in the testing requirement it says "Test that after typing to the input and pressing Enter key, the "onChange" prop is called with proper value". Unfortunately fireEvent.keyDown(inputElement, { key: "Enter", }); does not trigger the submit (browser does it automatically when we press enter on a focused input). If I were to write the test the way I wanted I would test "submit" event instead of "keydown", but the requirement was different.